build-tools/packages/build-cli/src/commands/report/codeCoverage.ts

```diff
@@ -113,6 +105,23 @@ export default class ReportCodeCoverageCommand extends BaseCommand<
 			token: flags.githubApiToken,
 		};
 
+		let skipBuildFailureOnRegression = false;
```

Comment:
Nit: thoughts on inverting this to be `shouldFailBuildOnRegression`? The "skip" part gives it some feeling of a double negation for me that makes me stop and think twice every time I see a condition with it.

Reply:
done